docs/contributing/README.md

```diff
@@ -0,0 +1,136 @@
+# Contributing to Carbon
+
+Thanks for contributing to Carbon! :1st_place_medal: :1st_place_medal:
+
+-   [Contributing to Carbon](#contributing-to-carbon)
+    -   [Code of Conduct](#code-of-conduct)
+    -   [Consumers](#consumers)
+    -   [Developers](#developers)
+        -   [Development](#development)
+            -   [Pulling in dependencies](#pulling-in-dependencies)
+            -   [Building the project](#building-the-project)
+        -   [Code Review](#code-review)
+            -   [Pull Request checklist](#pull-request-checklist)
+            -   [Commit message format](#commit-message-format)
+            -   [Pull Request](#pull-request)
+        -   [Release](#release)
+        -   [Deploying the site to GitHub](#deploying-the-site-to-github)
+    -   [Breaking changes](#breaking-changes)
+
+This project uses NPM for dependency management and provides NPM scripts to call on `Webpack` to run tasks on the project.
+To get started, you will need to checkout this project.
+
+## Code of Conduct
+
+Carbon adheres to the [JS Foundation Code of Conduct](https://js.foundation/community/code-of-conduct).
```

Comment:
Will we align this code of conduct with Terra's?

https://github.com/cerner/terra-core/blob/main/CODE_OF_CONDUCT.md